api/utils.py

```diff
@@ -1,7 +1,22 @@
+import json
 import random
 import time
 
 
-def naptime():
+def random_nap_time():
     sleep_time = random.randint(0, 5)
-    time.sleep(sleep_time)
\ No newline at end of file
+    time.sleep(sleep_time)
+
+
+def perform_evil_roulette():
```

Comment:
Again, I don't think this should've been abstracted out. It's pretty clear what's going on and creating an api to custom this is actually more cognitive load.